<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardEdit</class>
 <widget class="QDialog" name="CardEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑卡片</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="classifyHorizontalLayout">
     <item>
      <widget class="QLabel" name="c1Label">
       <property name="text">
        <string>一级分类：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c1LineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="c2Label">
       <property name="text">
        <string>二级分类：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c2LineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="c3Label">
       <property name="text">
        <string>三级分类：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c3LineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="wordHorizontalLayout">
     <item>
      <widget class="QLabel" name="wordLabel">
       <property name="text">
        <string>单词/短语/句子/语法：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wordLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="kanaHorizontalLayout">
     <item>
      <widget class="QLabel" name="kanaLabel">
       <property name="text">
        <string>假名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="kanaLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tagHorizontalLayout">
     <item>
      <widget class="QLabel" name="tagLabel">
       <property name="text">
        <string>标签：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tagLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="typeHorizontalLayout">
     <item>
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="typeLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="partOfSpeechHorizontalLayout">
     <item>
      <widget class="QLabel" name="partOfSpeechLabel">
       <property name="text">
        <string>词性：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="partOfSpeechLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="paraphraseHorizontalLayout">
     <item>
      <widget class="QLabel" name="paraphraseLabel">
       <property name="text">
        <string>释义：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="paraphraseTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="noteHorizontalLayout">
     <item>
      <widget class="QLabel" name="noteLabel">
       <property name="text">
        <string>笔记：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="noteTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CardEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CardEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
